src/core/wee-network.c

```diff
@@ -107,9 +107,21 @@ network_set_gnutls_ca_file ()
         ca_path2 = string_replace (ca_path, "%h", weechat_home);
         if (ca_path2)
         {
-            gnutls_certificate_set_x509_trust_file (gnutls_xcred, ca_path2,
-                                                    GNUTLS_X509_FMT_PEM);
+            int num_paths, i;
```

Comment:
It is a convention throughout WeeChat source to declare variables at the very beginning of a function.